pyi.py

```diff
@@ -508,8 +530,10 @@ def should_warn(self, code):
 Y014 = 'Y014 Default values for arguments must be "..."'
 Y015 = 'Y015 Attribute must not have a default value other than "..."'
 Y016 = "Y016 Duplicate union member"
+Y017 = "Y017 Only simple assignments allowed"
 Y090 = "Y090 Use explicit attributes instead of assignments in __init__"
 Y091 = 'Y091 Function body must not contain "raise"'
 Y092 = "Y092 Top-level attribute must not have a default value"
+Y093 = "Y093 Use typing_extensions.TypeAlias for type aliases"
```

Comment:
It seems like Y09* are used for disabled by default errors. Was the intention to add Y093 to that list?

Reply:
Yes, because we can't use it for typeshed yet (because of missing support in pytype I believe). We might have to rethink this convention though; the other Y09* checks can probably be safely enabled by default.

Reply:
Cool, maybe then let's make this Y018, assuming we want to turn this on once all type checkers have support.

I couldn't find any hits for Y09* error codes on grep.app and cs.github.com so we could probably easily change the other error codes too if we wanted.

Reply:
I'd like to land it as Y093 first; making a non-Y09* code disabled by default is likely complicated.

Reply:
Afaict there isn't any magic, it's just whatever is in `DISABLED_BY_DEFAULT` on L515, which this isn't. Sorry my initial comment could have been clearer.

Reply:
Not quite, there's also special casing in `should_warn` on line 479.

Reply:
Good catch, maybe that should just check for membership in DISABLED_BY_DEFAULT

Reply:
No, because it also needs to support `--select=Y09`. I'd rather keep the existing structure for this PR and revisit it when we address #75.

Reply:
Okay, makes sense. Maybe I'm being thick, but we still need to add `Y093` to `DISABLED_BY_DEFAULT` right (as per my first comment)?

Reply:
Good catch, applied that